src/models/serverless.ts

```diff
@@ -53,6 +53,22 @@ export interface ServerlessAzureProvider {
   armTemplate?: ArmTemplateConfig;
   keyVaultConfig?: AzureKeyVaultConfig;
   runtime: string;
+  functionRuntime?: FunctionRuntime;
+}
+
+export enum FunctionAppOS {
```

Comment:
Doesn't look like this is being used anywhere?  I'm assuming this will get used as Python support gets rolled in?